src/Discovery/test/HttpClients.Test/ResolvingHttpDelegatingHandler.cs

```diff
@@ -0,0 +1,16 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information.
+
+#pragma warning disable IDE0130 // Namespace does not match folder structure
+// ReSharper disable once CheckNamespace
+namespace Microsoft.Extensions.ServiceDiscovery.Http;
+#pragma warning restore IDE0130 // Namespace does not match folder structure
+
+internal sealed class ResolvingHttpDelegatingHandler : DelegatingHandler
```

Comment:
I wondered, but then assumed (and confirmed) the real type is `internal` and that's why this copy is here... We could add a note to eliminate future guessing, but that probably isn't _necessary_